services/github/src/api.js

```diff
@@ -29,24 +29,28 @@ function sanitizeGitHubField(field) {
 //
 // See https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pullrequestevent
 function getPullRequestDetails(eventData) {
+  const pr = eventData.pull_request;
+
+  // Since validation ensures head.repo exists, we can safely access it
+  // No need for fallback values anymore
```

Comment:
This comment implies that something existed here before :) 
Can we leave this function as it was before, since it's not changing anything in terms of functionality?